examples/basics/quickstart/README.md

```diff
@@ -0,0 +1,106 @@
+# Quickstart
+
+This is a simple Dynamo example showing how you can quickly get started deploying Large Language Models with Dynamo.
+
+## Components
+
+- [Frontend](../../../components/frontend/README) - A built-in component that launches an OpenAI compliant HTTP server, a pre-processor, and a router in a single process
+- [vLLM Engine](../../vllm/components/main.py) - A custom component that runs vLLM within the dynamo runtime
+
+```text
+Request Flow
+==================================================
+
+Users/Clients (HTTP)
+      │
+      ▼
+┌─────────────┐
+│  Frontend   │  HTTP API endpoint (Open-AI Style)
+└─────────────┘
+      │
+      ▼
+┌─────────────┐
+│ vLLM engine │  HTTP API endpoint (/generate)
+└─────────────┘
+```
+
+```mermaid
+---
+title: Request Flow
+---
+flowchart TD
+    A["Users/Clients<br/>(HTTP)"] --> B["Frontend<br/>HTTP API endpoint<br/>(OpenAI Style)"]
+    B --> C["vLLM engine<br/>HTTP API endpoint<br/>(/generate)"]
+```
+
+## Instructions
+
+There are three steps to deploy an llm with dynamo.
+
+### 1. Launch Engine
+
+```bash
+python3 dynamo/examples/vllm/components/main.py --model Qwen/Qwen3-0.6B
```

Comment:
This will be `python -m dynamo.vllm --model ...` as soon as the UX v2 work completes. It's done for llama.cpp, trtllm and sglang, but not yet vllm.

Reply:
I can swap it over to sglang to take advantage of what's working now

Reply:
Good idea.